mdfreader/mdf3reader.py

```diff
@@ -949,7 +949,12 @@ def read3(self, fileName=None, info=None, multiProc=False, channelList=None, \
 
                         for chan in channels: # for each recordchannel
                             recordName = buf[recordID]['record'].recordToChannelMatching[chan.recAttributeName]  # in case record is used for several channels
-                            temp = buf[recordID]['data'][recordName]
+                            try:
```

Comment:
Honestly, I would rather avoid adding try like this, especially for performance reason. Maybe I coudl help you to solve the issue than asked you to put this try statement (and the next one) ?